kitchen-tests/cookbooks/end_to_end/recipes/linux.rb

```diff
@@ -122,6 +122,7 @@
 include_recipe "::_ohai_hint"
 include_recipe "::_openssl"
 include_recipe "::_tests"
+include_recipe "::_linux_mount"
```

Comment:
Please rename this to _mount to keep the pattern of matching the resource name going. It's fine if we only use it on Linux for now. If we want to expand mount testing to other platforms we can add some platform logic there later.

Reply:
Agreed, I have updated this.